README.md

```diff
@@ -28,6 +28,26 @@ remotes::install_github("NOAA-FIMS/FIMS")
 library(FIMS)
 ```
 
+#### Fixing Fatal Error
+
+Windows users can expect to see some derivative of the following error message in their R session if they have not yet set some flags using {withr}.
+```
+Fatal error: can't write <xxx> bytes to section .text of FIMS.o: 'file too big
+```
+You can easily fix this by running
+```
+withr::local_options(pkg.build_extra_flags = FALSE)
+```
+This fix removes the debugger flag `-O0 -g` from being automatically inserted for certain devtools calls (eg. `devtools::load_all()`). Windows developers wanting to compile FIMS with the debugger turned on will need to run the above script in addition to manually modifying the [Makevars.win](https://github.com/NOAA-FIMS/FIMS/blob/doc-install/src/Makevars.win) file in the src directory to the following:
```

Comment:
* "eg." should be "e.g.,"
* add "call to PKG_CXXFLAGS in the" after "modifying the"